sdk-workflows/src/main/java/io/dapr/workflows/runtime/WorkflowRuntimeBuilder.java

```diff
@@ -70,5 +90,77 @@ public <T extends WorkflowActivity> void registerActivity(Class<T> clazz) {
     this.builder = this.builder.addActivity(
         new ActivityWrapper<>(clazz)
     );
+    String logTime = getLogTime();
+    this.logger.log(Level.INFO, logTime + " Registered Activity: " +  clazz.getSimpleName());
+    this.activities.add(clazz.getSimpleName());
+  }
+
+  /**
+   * Sets the log level for the workflow runtime builder.
+   */
+  private void setLogLevel() {
```

Comment:
In Java, log level and date format is done from the application's perspective. See the example here: https://github.com/dapr/java-sdk/blob/master/examples/src/main/resources/logback.xml

If you need this from the test's perspective, you can modify this class to have a private constructor that will receive the custom logger from the test.